matplotlibrc.template

```diff
@@ -628,7 +628,7 @@
                      ##         line (in pixels).
                      ##     - *randomness* is the factor by which the length is
                      ##         randomly scaled.
-#path.effects : []   ##
+#path.effects :
```

Comment:
Is this change necessary? If so, should one worry about it being necessary?

Reply:
Indeed, the template has been wrong for a while....

Reply:
This previously eval'ed to the "[]" string, which is just plain wrong.  Indeed, this is one of the motivations for dogfooding our own style template.